<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNXPCommandClass</class>
 <widget class="QDialog" name="CNXPCommandClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen2 - Set NXP Command</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Qrfe/images/qrfe.qrc">
    <normaloff>:/ams logos/mtiWindowIcon</normaloff>:/ams logos/mtiWindowIcon</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../Qrfe/images/qrfe.qrc">:/window images/keyImage</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set NXP Command</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>EPC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="tagIdEdit">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Access Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="inputMask">
      <string>HH-HH-HH-HH; </string>
     </property>
     <property name="text">
      <string>00-00-00-00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QTextEdit" name="infoEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Command:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="commandLineEdit">
     <item>
      <property name="text">
       <string>Set Read Protect 02-01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unset Read Protect 02-00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change EAS ON 01-01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change EAS OFF 01-00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trigger EAS Alarm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ChangeConfig 09-00</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Configuration:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="configLineEdit">
     <property name="inputMask">
      <string>HH-HH; </string>
     </property>
     <property name="text">
      <string>00-00</string>
     </property>
     <property name="cursorPosition">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
  <include location="../../../../../../../../Documents and Settings/Qrfe/images/qrfe.qrc"/>
  <include location="../../../Qrfe/images/qrfe.qrc"/>
 </resources>
 <connections/>
</ui>
